subprojects/docs/src/docs/userguide/authoring_maintainable_builds.adoc

```diff
@@ -141,7 +141,7 @@ include::sample[dir="userguide/bestPractices/taskDefinition/kotlin", files="buil
 ====
 
 [[sec:improving_task_discoverability]]
-== Improving task discoverability
+== Improve task discoverability
```

Comment:
The nature of this chapter is different from others. These are best practices that we want people to follow, so I prefer the stronger imperative. It doesn't imply that readers _must_ follow these directives, but what we are saying is that "if you want to apply best practice, do this, do that".

I would find a style guide helpful myself, because I'm often inconsistent with myself. It's just more work and harder than it seems.

Reply:
I don't have a strong opinion here, so feel free to keep your changes.